crates/xline/src/conflict/tests.rs

```diff
@@ -0,0 +1,221 @@
+use std::sync::Arc;
+
+use curp::{rpc::ProposeId, server::conflict::CommandEntry};
+use curp_external_api::conflict::{ConflictPoolOp, SpeculativePoolOp, UncommittedPoolOp};
+use xlineapi::{
+    command::{Command, KeyRange},
+    AuthEnableRequest, AuthRoleAddRequest, DeleteRangeRequest, LeaseGrantRequest,
+    LeaseRevokeRequest, PutRequest, RequestWrapper,
+};
+
+use crate::conflict::{
+    spec_pool::ExclusiveSpecPool,
+    uncommitted_pool::{ExclusiveUncomPool, KvUncomPool, LeaseUncomPool},
+};
+
+use super::spec_pool::{KvSpecPool, LeaseSpecPool};
+
+#[test]
+fn kv_sp_operations_is_ok() {
```

Comment:
nit: it should be `kv_sp_operations_are_ok`

Reply:
fixed